ckcp/openshift_dev_setup.sh

```diff
@@ -265,53 +248,37 @@ patches:
-    # Check if workspace "demo" is ready. This is a workaround, because commands executed directly after creating workspace with "--enter" option occasionally fail
-    while [[ $(KUBECONFIG="$KUBECONFIG_KCP" kubectl get workspace -o jsonpath='{.items[0].status.phase}') != "Ready" ]]; do
-      echo -n "."
-      sleep 5
-    done
-  fi
-  KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workspaces use demo >/dev/null 2>&1
-  echo "OK"
-
-  echo -n "  - Workloadcluster pipeline-cluster registration: "
-  if ! KUBECONFIG="$KUBECONFIG_KCP" kubectl get WorkloadCluster local >/dev/null 2>&1; then
-    (
-      kcp_image_tag="$(yq -e '.images[0].newTag' "$ckcp_dev_dir"/kustomization.yaml)"
-      cr_string="$(IFS=,; echo "${CR_TOSYNC[*]}")"
-      KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workload sync local \
-      --syncer-image ghcr.io/kcp-dev/kcp/syncer:"$kcp_image_tag" \
-      --resources "$cr_string" > "$kube_dir/syncer.yaml"
-      kubectl apply -f "$kube_dir/syncer.yaml" >/dev/null 2>&1
-      # Wait until Syncer pod is available
-      SYNCER_NS_ID="$(kubectl get ns -l workload.kcp.io/workload-cluster=local -o json | jq -r '.items[0].metadata.name')"
-      kubectl rollout status deployment/kcp-syncer -n "$SYNCER_NS_ID" --timeout=90s >/dev/null 2>&1
-      rm -rf "$kube_dir/syncer.yaml"
-    )
-  fi
-  echo "OK"
+  # Workaround to prevent the creation of the creation of a new workspace until KCP is ready.
+  # This fixes `error: creating a workspace under a Universal type workspace is not supported`.
+  while ! KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workspace create "dummy" --ignore-existing >/dev/null; do
+    sleep 5
+  done
 
-  # Register the KCP cluster into ArgoCD
-  echo -n "  - KCP cluster registration to ArgoCD: "
-  if ! KUBECONFIG="$KUBECONFIG_MERGED" argocd cluster get kcp >/dev/null 2>&1; then
-    KUBECONFIG="$KUBECONFIG_MERGED" argocd cluster add "workspace.kcp.dev/current" --name=kcp  --system-namespace default --yes >/dev/null 2>&1
-  fi
-  echo "OK"
+  echo "  - Setup kcp access:"
+  "$SCRIPT_DIR/../images/access-setup/content/bin/setup_kcp.sh" \
+    --kubeconfig "$KUBECONFIG_KCP" \
+    --kcp-workspace "$kcp_workspace" \
+    --work-dir "$WORK_DIR"
+  KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/ckcp-ckcp.default.compute.kubeconfig"
 }
 
-install_openshift_pipeline() {
-  APP="argocd"
+install_compute(){
+  echo "  - Setup compute access:"
+  "$SCRIPT_DIR/../images/access-setup/content/bin/setup_compute.sh" \
+    --kubeconfig "$KUBECONFIG" \
+    --work-dir "$WORK_DIR"
 
-  echo -n "  - openshift-pipeline application: "
-  kubectl apply -f "$GITOPS_DIR/$APP/$APP.yaml" --wait >/dev/null 2>&1
-  argocd app wait pipelines-service tektoncd --timeout=90 >/dev/null 2>&1
-  echo "OK"
+  echo "  - Deploy compute:"
+  "$SCRIPT_DIR/../images/cluster-setup/install.sh" --workspace-dir "$WORK_DIR"
+
+  echo "  - Setup Pipelines as Code:"
+  GITOPS_REPO="https://example.git.com/my/repo" GIT_TOKEN="placeholder_token" \
```

Comment:
Could you please put a comment before this line with the reason to call the script with the dummy values?  something like

#Passing dummy values to the parameters of the pac/setup.sh script because we only want to install the runner side of resources.